.github/dependabot.yml

```diff
@@ -17,9 +17,11 @@ updates:
     allow:
       - dependency-type: "development"
     ignore:
+      - dependency-name: "phpstan/*"
       - dependency-name: "phpunit/phpunit"
         versions: [ ">= 10.0.0" ]
       - dependency-name: "rawr/cross-data-providers"
+      - dependency-name: "rector/rector"
```

Comment:
I presume that the GitHub action is unable to cope with `OR` depencies.  This can be revisted later if and when it is able.